microsoft-edge/webview2/release-notes/index.md

```diff
@@ -122,9 +119,6 @@ The following APIs have been promoted to Stable and are now included in this Rel
    * `Client`
    * `Nowhere`
 
-* `CoreWebView2ProcessFailedEventArgs` Class:
-   * [CoreWebView2ProcessFailedEventArgs.FailureSourceModulePath Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2processfailedeventargs?view=webview2-winrt-1.0.2420.47&preserve-view=true#failuresourcemodulepath)
```

Comment:
May I ask why is this removed?

Reply:
There's an error in March relnotes for Release, Promotions to stable: 
https://learn.microsoft.com/microsoft-edge/webview2/release-notes/#promotions

So, in .NET tab & WinRT tab, delete this item here, which is dup'd from the subsequent Promotion entry: 
CoreWebView2ProcessFailedEventArgs.FailureSourceModulePath Property

Rendered PR: 
https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/?branch=pr-en-us-3096&tabs=dotnetcsharp#promotions -- item has been removed from first tab-set, b/c it's properly already in the 2nd tab-set.